app/views/pages/_editor_guide_text.html.erb

```diff
@@ -213,7 +230,8 @@
     <h3><strong>Parsing Liquid Tags as a Code Example</strong></h3>
     <p>To parse Liquid tags as code, simply wrap it with a single backtick or triple backticks.</p>
     <p><code>`{% mytag %}{{ site.SOMETHING }}{% endmytag %}`</code></p>
-    <p>One specific edge case is with using the <code>raw</code> tag. To properly escape it, use this format:</p>
+    <p>One specific edge case is with using the <code>raw</code> tag. To properly escape it, use this format:
```

Comment:
Some indentations fixes are strange (autocorrect?). I think the initial format was good enough, or assume indentation to be:
```html
<p>
  One specific edge case is with using the <code>raw</code> tag. To properly escape it, use this format:
</p>
```
However, I wouldn't like to focus too much on such small details. 
What do you think?

Reply:
I think this was my IDE doing a line wrap.
For me it is good either way only downside might be that there are other cases like this that have already been merged.